test/perf/kernel/stockcorr.jl

```diff
@@ -4,7 +4,6 @@
 
 #STOCKCORR - The original, unoptimised code that simulates two correlated assets
 function stockcorr()
-
     ## Correlated asset information
```

Comment:
Please don't delete blank lines preceding comments in cases like this where the author has clearly taken some time and care with manual ASCII layout to in an attempt to make the code easy to understand. A comment is the typographical equivalent of a heading, or the beginning of a paragraph, and should be clearly separated from whatever precedes it.